package-lock.json

```diff
@@ -1,8 +1,14107 @@
 {
   "name": "echarts",
   "version": "5.2.2",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Seems the format of your package-lock.json is different from mine

Reply:
That's because I'm using a newer version(v8) of npm. The lockfile v2 is generated since npm v7 and it is backward compatible with npm v6.

See also
- https://github.com/npm/cli/blob/latest/docs/content/configuring-npm/package-lock-json.md#lockfileversion
- https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile

Reply:
Get stucked at installing the dependencies after upgraded to npm@8. Perhaps we need to consider switching to pnpm or yarn.

Reply:
It keeps giving me `unable to resolve dependency tree` error

Reply:
> Perhaps we need to consider switching to pnpm or yarn.

I second that. It's definitely better.

Reply:
Let's use the v1 lockfile. I will revert the changes. You can commit your package.json later.

Reply:
Thanks for giving me this chance.